src/map/atcommand.c

```diff
@@ -8366,6 +8366,36 @@ void atcommand_commands_sub(struct map_session_data* sd, const int fd, AtCommand
 	dbi_destroy(iter);
 	clif->message(fd,line_buff);
 
+	if (atcommand->binding_count) {
```

Comment:
according to the code style guidelines, this should be `atcommand->binding_count > 0`